src/WebUI/Common/KubeFilterBar.tsx

```diff
@@ -52,7 +52,7 @@ export class KubeFilterBar extends BaseComponent<IFilterComponentProperties, {}>
                         hideClearButton={false}
                         getPickListItems={this.props.pickListItemsFn}
                         getListItem={this.props.listItemsFn}
-                    />
+                    />*/}
```

Comment:
PicklistFilterBarItem is not available in azure-devops-ui 151 package, so commenting out this code till we fix it

Reply:
Replaced with DropdownFilterBar